### ISSUE TITLE
nothing works even the stop button dont work

### ISSUE THREAD

**Issue by clownnut** [finndev/PokeBuddy#719]
the old one work but keep getting access token expired error and keep farming the same 4 pokemons stop over and over and over. so i just download the newest one from here, worse then before. i am still getting the access token expired error. now no pokestop show on the map, no pokemon catch. even the stop botton doesnt work. when you click the stop button, the bot just keep on moving around.

**Comment by Thorben24** [comment]
If there are no pokestops on the map it is an IP-Ban. You have to change your IP adress in this case to solve this problem. 95% of the providers use dynamic ip adressing, so a relog / restart of your router should renew your ip adress.

**Comment by clownnut** [comment]
that's going to be a problem. comcast dont have dynamic ip addressing....

**Comment by forestlin98** [comment]
Me too man, i cant see any pokestop and stop button is not working

**Comment by Thorben24** [comment]
Well in this case it is a big problem. Do you have all the time the same ip adress or is it switching somewhen? Usually the IP ban will only happen if you bot / play more than 1 account at the same time. I dont know how long it will be banned, but i think just for some hours. If you need to bot now use a good VPN or a proxy server as long as your ip is banned. You can also just use a vpn for logging in and switching back to your home adress after it. Here is a great tutorial: http://www.ownedcore.com/forums/pokemon-go/pokemon-go-hacks-cheats/570331-quick-ip-unban-vpn-ip-no-ban.html

**Comment by clownnut** [comment]
yeah, comcast dont change ip address. only once a blue moon. i am at my place over 10 years only 1 ip address change in that time frame. i did play 2 accounts at the same time. didnt know that lol. well, we will see in few hours. if not, i had my fun. thanks for the link good info.

**Comment by Thorben24** [comment]
As i know meanwhile by myself and some other guys, the ip ban will be solved after 1-6 hours. To go safe, just dont login for the next 2-3 hours, if its still banned wiat straight for 6 hours.

**Comment by clownnut** [comment]
cool thanks for the tips. now this is why you shouldnt touch your computers before the coffee. i normally dont run 2 bots at the same time lol.

**Comment by clownnut** [comment]
cool thanks for the tips. now this is why you shouldnt touch your computers before the coffee. i normally dont run 2 bots at the same time lol.

**Comment by Thorben24** [comment]
Hehe true, but many guys, include me had this problem, too. They changed something last night (Pokemon Go or maybe badly the devs by themself that they detect this.

**Comment by clownnut** [comment]
i am surprised this is in place in the first place, but i guess they didnt think pokemon go will be this big or they just too busy to watch the money rolling in.

**Comment by clownnut** [comment]
looks like the ip ban is over about 1hr, at least on the account i dont care about lol. just a testing account.

**Comment by clownnut** [comment]
still have the token expired error and still only visit 4 pokestops over and over

**Comment by Thorben24** [comment]
Strange. Try to change the coordinates. Whats your walking speed? Do you have Farm Pokestops and Catch Pokemons activated?

**Comment by clownnut** [comment]
i set the speed at 20m/s
yes on the farm pokestops and catch pokemons.
i tried to move the coordinates around. dont help.
as far as i can tell, i think the token expired error is causing the pokestops issue. 
the token error happen every 10 sec. it take the bot about 10 second to visit the 3-4 pokestops(they are very close) then the token error happen, instead of moving to other new pokestops the bot just revisit all the pokestops it just visited.

**Comment by Thorben24** [comment]
How much XP you get when you farm a pokestop?

**Comment by clownnut** [comment]
50(90% of the time)

**Comment by Thorben24** [comment]
And you tested a location who not faced this 4 pokestops? For me it seems like they are broken or something. Never seen an issue like that.

**Comment by clownnut** [comment]
yes, i moved it around. i mostly farm San Fran bar area. i tried fort mason area/north beach(3000+ft apart). it seem after the token error, the bot want to visit the closest pokestop, so it just keep running around in circle.  but i did pick the spots where there are 4-5 pokestops within 200m or less.
but i will try some area the pokestops are a bit more space out see if that will help.

still dont solve the token expired error....

**Comment by Thorben24** [comment]
are you using google or ptc login?

If google, create a app-pasword in your google account and replace in pokebuddy after that the google account password with the app-password you will get by google: https://security.google.com/settings/security/apppasswords

**Comment by clownnut** [comment]
google login.
i did the app password and replace the normal login password with the new app password.
still have the token error, instead of every 10 secs now is every 11 secs. lol

**Comment by Thorben24** [comment]
i dont get it lol. did you try another account?

**Comment by clownnut** [comment]
i havent, will try that later today.

**Comment by clownnut** [comment]
no change still getting token error on 2 google account.
